include/caffe/layers/base_data_layer.hpp

```diff
@@ -68,15 +68,16 @@ class BasePrefetchingDataLayer :
       const vector<Blob<Dtype>*>& top);
 
   // Prefetches batches (asynchronously if to GPU memory)
-  static const int PREFETCH_COUNT = 3;
+  static const int PREFETCH_COUNT = 4;  // same as proto
```

Comment:
If this is the same as the proto def, then drop this. No need to have fragile duplication.